WebDriverAgentMac/WebDriverAgentLib/Commands/AMWindowCommands.m

```diff
@@ -32,21 +32,21 @@ + (NSArray *)routes
 {
   return [self performClickWithButtonId:@"_XCUI:FullScreenWindow"
                                 request:request
-                           errorMessage:[NSString stringWithFormat:@"%@ window cannot be maximized because the correponding button is not available", request.session.currentApplication.description]];
+                            errorMessage:[NSString stringWithFormat:@"%@ window cannot be maximized because the corresponding button is not available", request.session.currentApplication.description]];
 }
 
 + (id<FBResponsePayload>)handleFullscreenWindow:(FBRouteRequest *)request
 {
   return [self performClickWithButtonId:@"_XCUI:FullScreenWindow"
                                 request:request
-                           errorMessage:[NSString stringWithFormat:@"%@ window cannot be put into fullscreen mode because the correponding button is not available", request.session.currentApplication.description]];
+                            errorMessage:[NSString stringWithFormat:@"%@ window cannot be put into fullscreen mode because the corresponding button is not available", request.session.currentApplication.description]];
```

Comment:
```suggestion
                           errorMessage:[NSString stringWithFormat:@"%@ window cannot be put into fullscreen mode because the corresponding button is not available", request.session.currentApplication.description]];
```